internal/generator/vector/filter/apiaudit/filter.go

```diff
@@ -0,0 +1,53 @@
+package apiaudit
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"text/template"
+
+	loggingv1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func init() {
+	utilruntime.Must(auditv1.AddToScheme(scheme.Scheme))
```

Comment:
I like having it close to the point of use so that unit tests can run, it's' more obvious why its needed etc.
We have some of these central  in main.go but we do have a couple of other examples that are close to the point of use. We also have a bunch of places where we have to add this in test code because it's not done at the point of use, which I don't like so much. There's no cost if we init the same types twice.